frontend/components/ViewingModal.vue

```diff
@@ -73,20 +82,54 @@
           </div>
         </div>
         <div class="modal-footer">
-          <button
-              type="button"
-              class="btn btn-default"
-              data-dismiss="modal"
-          >
-            Close
-          </button>
-          <button
-              v-if="isImage && isEditable"
-              type="button"
-              class="btn btn-primary"
-          >
-            Save changes
-          </button>
+          <div class="row">
+            <div
+                v-if="isImage && isEditable"
+                class="col-sm-10 text-left"
+            >
+              <button
+                  type="button"
+                  class="btn btn-default"
+                  @click="rotation -= 90"
+              >
+                <i class="fas fa-undo" />
+              </button>
+              <button
+                  type="button"
+                  class="btn btn-default"
+                  @click="rotation += 90"
+              >
+                <i class="fas fa-redo" />
+              </button>
+              <button
+                  type="button"
+                  class="btn btn-primary"
+                  :disabled="!(rotation % 360)"
```

Comment:
N'est-il pas plus intéressant d'extraire ces conditions dans des fonctions du composants afin de pouvoir faciliter l'évolutivité/compréhension? 

- rotateClockwise()
- rotateCounterClockwise()
- haveAnyRotationApplied()

Reply:
Oui en effet je vais rajouter quelques `computed` pour améliorer la compréhension du code, je pense que `rotation += 90` et `rotation += -90` sont assez simples/parlants mais les morceaux comme `this.rotation % 180` le sont moins